core/src/main/resources/lib/layout/breadcrumbs.css

```diff
@@ -51,6 +51,7 @@
 
 #breadcrumbs LI:hover, A.breadcrumbBarAnchor.mouseIsOverMenuSelector {
   background-color: #e0e4dc;
+  background-color: var(--breadcrumbs-item-bg-color--hover);
```

Comment:
Do we need a double dash in `--hover`?

Reply:
the other variables have them to indicate that they are an altered state from the default variable

https://github.com/jenkinsci/jenkins/blob/master/war/src/main/less/abstracts/colors.less#L117